packages/boot/test/bootstrapTests/test-orchestration.ts

```diff
@@ -120,4 +121,74 @@ test.serial('stakeAtom', async t => {
     matches(account, M.remotable('ChainAccount')),
     'account is a remotable',
   );
+
+  const atomBrand = await EV(agoricNames).lookup('brand', 'ATOM');
+  const atomAmount = AmountMath.make(atomBrand, 10n);
+
+  const res = await EV(account).delegate('cosmosvaloper1test', atomAmount);
+  t.is(res, 'Success', 'delegate returns Success');
+});
+
+test.serial('stakeAtom wallet driver', async t => {
+  const { agoricNamesRemotes } = t.context;
+
+  const wd = await t.context.walletFactoryDriver.provideSmartWallet(
+    'agoric1testStakAtom',
+  );
+
+  await wd.executeOffer({
+    id: 'request-account',
+    invitationSpec: {
+      source: 'agoricContract',
+      instancePath: ['stakeAtom'],
+      callPipe: [['makeCreateAccountInvitation']],
+    },
+    proposal: {},
+  });
+
+  const current = await wd.getCurrentWalletRecord();
+  const latest = await wd.getLatestUpdateRecord();
+  console.log({ current, latest });
+  t.like(current, {
+    offerToPublicSubscriberPaths: [
+      // TODO publish something useful
+      ['request-account', { account: 'published.stakeAtom' }],
+    ],
+  });
+  t.like(latest, {
+    status: { id: 'request-account', numWantsSatisfied: 1 },
+  });
+
+  const { ATOM } = agoricNamesRemotes.brand;
+  ATOM || Fail`ATOM missing from agoricNames`;
+
+  await t.notThrowsAsync(
```

Comment:
this doesn't guarantee it worked. consider checking vstorage

Reply:
Updated